Makefile

```diff
@@ -1,9 +1,13 @@
-DB2HOME=${GOPATH}/src/github.com/ibmdb/clidriver
+ifdef DB2_HOME
+	DB2HOME=${DB2_HOME}
+else
+	DB2HOME=${GOPATH}/src/github.com/ibmdb/clidriver
+	export LD_LIBRARY_PATH=${DB2HOME}/lib
+endif
 export GO15VENDOREXPERIMENT=1
 export GO111MODULE=on
 export CGO_CFLAGS=-I${DB2HOME}/include
 export CGO_LDFLAGS=-L${DB2HOME}/lib
-export LD_LIBRARY_PATH=${DB2HOME}/lib
```

Comment:
Given that DB2_HOME exists, I'm concerned about whether we can still compile successfully with the missing "export LD_LIBRARY_PATH=${DB2HOME}/lib" in this context.

Reply:
> yes, if DB2_HOME there, then LD_LIBRARY_PATH should be configured already, you donnot need to rely on third party driver if so.
In your environment, it may be the case that setting DB2_HOME alone is insufficient to ensure that LD_LIBRARY_PATH is configured correctly, which could potentially lead to compilation failures for others.